addons/account/models/account_move.py

```diff
@@ -2401,6 +2412,32 @@ def _validate_taxes_country(self):
                     raise ValidationError(_("This entry contains taxes that are not compatible with your fiscal position. Check the country set in fiscal position and in your tax configuration."))
                 raise ValidationError(_("This entry contains one or more taxes that are incompatible with your fiscal country. Check company fiscal country in the settings and tax country in taxes configuration."))
 
+    def _validate_tax_alignment_with_fpos(self):
+        """
+        Method to validate whether the tax_ids on the move_lines are aligned with the fiscal position of the move.
+
+        Returns:
+            bool: `True` if the taxes on the move_lines aligned with the fiscal position, otherwise `False`.
+        """
+
+        self.ensure_one()
+        line_ids = self.invoice_line_ids.filtered(
+            lambda line: not any(tax in line.tax_ids for tax in line._get_computed_taxes())
+        )
+        if line_ids:
+            message = "Taxes are not aligned with Fiscal Position ({}) set in other info tab".format(
+                self.fiscal_position_id.name
+            )
+            return {
+                'taxes_unaligned_with_fp': {
+                    'level': 'warning',
+                    'message': message,
+                    'action_text': _("View Journal Items(s)"),
+                    'action': line_ids._get_records_action(name=_("Journal Items(s)")),
```

Comment:
With this I found one more issue, if it's a single line then a form view would be open up better to pass some para to restrict the form view.
```suggestion
                    'action': line_ids._get_records_action(name=_("Journal Items(s)"), domain=[('id', 'in', line_ids.ids)], views=[(False, 'list')])
```

Reply:
![image](https://github.com/user-attachments/assets/239d326f-06ce-4f01-852b-c1b289df409b)

@hamo-odoo According to the PO, we should show only the warning message and remove the `View Journal(s)` button since mass editing is not possible. Additionally, even in the form view of the entry, tax fields are read-only and cannot be edited. Therefore, displaying just the warning message seems sufficient, And let users edit taxes directly on the account lines of the move only. What do you think? :thinking:

Reply:
Not being able to edit here is because of view_id (else it might give a unspecified view). You can simple pass it with a view_id (I have updated my above suggestion with view_id). or you can even create a new view for it if required, which might only display the taxes.

> According to the PO, we should show only the warning message and remove the View Journal(s) button since mass editing is not possible.

@jva-odoo What do you think about it?